packages/accounts/src/msca/client.ts

```diff
@@ -77,3 +102,59 @@ export async function createMultiOwnerModularAccountClient({
     .extend(multiOwnerPluginActions)
     .extend(accountLoupeActions);
 }
+
+export function createMultisigModularAccountClient<
+  TChain extends Chain | undefined = Chain | undefined,
+  TSigner extends SmartAccountSigner = SmartAccountSigner
+>(
+  args: CreateMultisigModularAccountClientParams<Transport, TChain, TSigner>
+): Promise<
+  SmartAccountClient<
+    CustomTransport,
+    Chain,
+    MultisigModularAccount<TSigner>,
+    MultisigPluginActions<MultisigModularAccount<TSigner>> &
+      PluginManagerActions<MultisigModularAccount<TSigner>> &
+      AccountLoupeActions<MultisigModularAccount<TSigner>>,
+    SmartAccountClientRpcSchema,
+    MultisigUserOperationContext
+  >
+>;
+
+export async function createMultisigModularAccountClient({
+  account,
+  transport,
+  chain,
+  ...clientConfig
+}: CreateMultisigModularAccountClientParams): Promise<
+  SmartAccountClient<
+    Transport,
+    Chain,
+    MultisigModularAccount<SmartAccountSigner>,
+    {},
+    SmartAccountClientRpcSchema,
+    MultisigUserOperationContext
+  >
+> {
+  // todo: load in the threshold here?
```

Comment:
is this relevant still? i'm not sure there's much utility to it since we'll always want the `getThreshold` function to return the onchain one if the contract is live

Reply:
deleted the comment